components/inputs/input-text.js

```diff
@@ -393,6 +420,11 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 		return 'text';
 	}
 
+	_handleAfterSlotChange(e) {
```

Comment:
I took the opportunity to rename handlers so we have consistent name, at least within this file.